packages/stark-core/src/modules/settings/services/settings.service.intf.ts

```diff
@@ -1,4 +1,9 @@
+import { InjectionToken } from "@angular/core";
+
 export const starkSettingsServiceName: string = "StarkSettingsService";
```

Comment:
Do we still need to export this const ? 
Maybe we should be exhaustive with our export in barrels because I've the feeling this serviceName const is uses only inside the service itself. What do you think about this ?

Reply:
That's true, it is only used inside every service... we could leave them out from the barrels if you want. Could you maybe create an issue for this?

Reply:
Thanks 😃